nixos/modules/programs/sway.nix

```diff
@@ -104,6 +103,15 @@ in {
         Extra packages to be installed system wide.
       '';
     };
+
```

Comment:
This options is pretty redundant/confusing. If you are trying to make a default option configurable you can simply set the value using `mkDefault`, see `programs.dconf.enable = mkDefault true;` just below.

Reply:
Done! Thanks for pointing that out